app/fonts/stylesheets/home.scss

```diff
@@ -0,0 +1,21 @@
+@font-face {
+    font-family: 'fairviewregular';
+    src: url('fairview_regular-webfont.eot');
+    src: url('fairview_regular-webfont.eot') format('embedded-opentype'),
+         url('fairview_regular-webfont.woff2') format('woff2'),
+         url('fairview_regular-webfont.woff') format('woff'),
+         url('fairview_regular-webfont.ttf') format('truetype'),
+         url('fairview_regular-webfont.svg') format('svg');
+    font-weight: normal;
+    font-style: normal;
+}
+
+.landing-title {
+  color: red;
```

Comment:
Color `red` should be written in hexadecimal form as `#ff0000`<br>Color literals like `red` should only be used in variable declarations; they should be referred to via variable everywhere else.